Support for Word 2016

igormf opened the pull request:
Added "SafePtr" modifier to macros, so Word 2016 will allow all functions

arnaudroques commented:
Thanks for your contribution.
However, we are uncomfortable about directly integrate a binary .dotm file without knowing the diffs
It would be nice if you could put in your commit the modifications done in module files https://github.com/plantuml/word-template/tree/master/module
Thanks again

igormf commented:
You are right, thought one could see straight into Visual Basic for Word. Just moved folders around so you can see differences in code. It's actually pretty simple, I've just added mandatory PtrSafe (without actually getting into the details) and worked straight out. Thanks for your great work.
